src/library/scala/collection/IterableOnce.scala

```diff
@@ -819,9 +829,10 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
    *  @tparam B      the type of the elements of the array.
    *  @return        the number of elements written to the array
    *
-   *  $willNotTerminateInf
+   *  @note    Reuse: $consumesIterator
    */
-  def copyToArray[B >: A](xs: Array[B]): Int = copyToArray(xs, 0)
+  @deprecatedOverriding("This should always forward to the 3-arg version of this method", since = "2.13.4")
+  @inline def copyToArray[B >: A](xs: Array[B]): Int = copyToArray(xs, 0, Int.MaxValue)
```

Comment:
Isn't inline at odds with deprecatedOverriding?

Reply:
possibly? I'm not sure.

I mean, the worst that can happen is it doesn't inline it. but it might be able to for leaf subtypes that don't override it? idk

Reply:
it seems it is not allowed

Reply:
> might be able to for leaf subtypes that don't override it? idk

it can do that

```
$> s -opt:l:inline '-opt-inline-from:**' -Vinline _
scala> :pa

class A { @inline def foo = 0 }
final class B extends A
def bar = (new B).foo

// Exiting paste mode, now interpreting.

Inlining into $line3/$read$$iw.bar: inlined $line3/$read$$iw$A.foo (the callee is annotated `@inline`). Before: 10 ins, after: 27 ins.
```